display/type340.c

```diff
@@ -670,27 +674,27 @@ static const unsigned char chars[128][5] = {
     { 0000, 0000, 0000, 0000, 0000 },   /* 40 space */
     { 0000, 0000, 0372, 0000, 0000 },   /* 41 ! */
     { 0000, 0340, 0000, 0340, 0000 },   /* 42 " */
-    { 0050, 0376, 0050, 0376, 0050 },   /* 43 # */
+    { 0100, 0200, 0100, 0040, 0100 },   /* 43 ~ */
     { 0144, 0222, 0376, 0222, 0114 },   /* 44 $ */
     { 0306, 0310, 0220, 0246, 0306 },   /* 45 % */
     { 0154, 0222, 0156, 0004, 0012 },   /* 46 & */
-    { 0000, 0000, 0300, 0340, 0000 },   /* 47 ' */
-    { 0070, 0104, 0202, 0000, 0000 },   /* 50 ( */
+    { 0020, 0070, 0124, 0020, 0020 },   /* 47 right arrow */
+    { 0010, 0004, 0376, 0004, 0010 },   /* 50 down arrow */
     { 0000, 0000, 0202, 0104, 0070 },   /* 51 ) */
-    { 0124, 0070, 0174, 0070, 0124 },   /* 52 * */
-    { 0020, 0020, 0174, 0020, 0020 },   /* 53 + */
-    { 0000, 0014, 0016, 0000, 0000 },   /* 54 , */
-    { 0020, 0020, 0020, 0020, 0020 },   /* 55 - */
-    { 0000, 0006, 0006, 0000, 0000 },   /* 56 . */
+    { 0100, 0040, 0020, 0010, 0004 },   /* 52 \ */
+    { 0376, 0202, 0202, 0000, 0000 },   /* 53 [ */
+    { 0000, 0000, 0202, 0202, 0376 },   /* 54 ] */
+    { 0020, 0154, 0202, 0000, 0000 },   /* 55 { */
+    { 0000, 0000, 0202, 0154, 0020 },   /* 56 } */
     { 0004, 0010, 0020, 0040, 0100 },   /* 57 / */
-    { 0174, 0212, 0222, 0242, 0174 },   /* 60 0 */
+    { 0002, 0002, 0002, 0002, 0002 },   /* 60 _ */
     { 0000, 0102, 0376, 0002, 0000 },   /* 61 1 */
-    { 0116, 0222, 0222, 0222, 0142 },   /* 62 2 */
+    { 0000, 0000, 0376, 0000, 0000 },   /* 62 | */
     { 0104, 0202, 0222, 0222, 0154 },   /* 63 3 */

```

Comment:
I think we should just remove all the duplicates from the "shifted" character set.  They were only there as place-holders.

Maybe replace them with 0376,  0376,  0376,  0376,  0376,  0376 (black square) to indicate that it's a (black) hole in our knowledge?

Did you add a circumflex?  Given that there are right and down arrows, and the age of the hardware, up-arrow might be more likely....

Reply:
I was thinking of the original ASCII, which had up-arrow.....  The PDP-6 was certainly at the cusp between six-bit and ASCII, I forget where in the 18-bit lineage the change occurred (maybe between the PDP-4 and PDP-7?)

Reply:
Thanks, I'll replace the unknown characters.

Yes, I added circumflex.  Good point; it's not unlikely there was an up arrow instead.  But if so, I think the underscore should be a left arrow.